Makefile

```diff
@@ -130,7 +130,7 @@ endif
 
 .PHONY: test-cli
 test-cli: container
```

Comment:
Can you rename target to test-image? Update also target test-version on line 288

Reply:
Do you mean rename or duplicate such that there is a `test-cli` and `test-image`?

Reply:
here we  should be ok with rename.